src/openalea/widgets/plantgl.py

```diff
@@ -212,6 +212,25 @@ def PlantGL(pglobject, plot=None, group_by_color=True, property=None, side='fron
     #plot.colorbar_object_id = randint(0, 1000)
     return plot
 
+def PlantGL_Mesh(pglobject, plot=None, group_by_color=True, property=None, side='front'):
```

Comment:
I do not completly understand.
This is already done by the PlantGL function, isn't it?
Why do you need another function?